tools/api-markdown-documenter/src/test/test-data/simple-suite-test/test-suite-a.api.json

```diff
@@ -240,7 +240,7 @@
 						{
 							"kind": "Property",
 							"canonicalReference": "test-suite-a!TestAbstractClass#abstractPropertyGetter:member",
-							"docComment": "/**\n * A test abstract getter property.\n */\n",
+							"docComment": "/**\n * A test abstract getter property.\n *\n * \\@escapedTag\n */\n",
```

Comment:
Note for reviewers: This updates the test input comment to include escaped contents. The above test output changes are reflections of this.